scipy/stats/__init__.py

```diff
@@ -343,16 +343,13 @@
 
 
 Masked statistics functions
-===========================
+---------------------------
 
 .. toctree::
 
    stats.mstats
 
 
-Other statistical functionality
```

Comment:
Let's just leave this alone. Seems out of scope, and it is debatable, so let's not slow things down.